.github/workflows/deploy-reliability-platform.yml

```diff
@@ -44,7 +57,7 @@ jobs:
       resourceGroupName: ${{ needs.fetch-targets.outputs.resourceGroupName }}
       distroName: ${{ needs.fetch-targets.outputs.distroName }}
       cpuThreshold: 50
-      repeatForXHours: 2
+      repeatForXHours: 1
```

Comment:
Is the plan to increase this number later?

Reply:
yes, this will make it work temporarily until i figure out how to get it to work passed the SAS token expiration (2hrs)